src/index.ts

```diff
@@ -8,26 +8,39 @@ import { Scenario } from './model/Scenario';
 import { TestResult } from './model/TestResult';
 import { loadAllScenarios, loadScenariosById } from './scenarioLoading';
 import { loadYamlConfiguration } from './yamlParsing';
+import { stringify } from 'querystring';
 
 const RESULTS: Map<string, TestResult[]> = new Map();
 
-const NUM_PARALLEL_RUNS = 10;
-
 let OUT_DIR = '';
 
 // increase the pixel-size (width/height) limit of PlantUML (the default is 4096 which is not enough for some diagrams)
 process.env.PLANTUML_LIMIT_SIZE = '16384';
 
-export const runScenario = (
-    scenarioPath: string,
+interface RunConfiguration {
+    numberOfScenariosRunInParallel?: number;
+    environmentNameToBeUsed?: string;
+}
+
+export const runMultipleSceanriosWithConfig = (
```

Comment:
Wouldn't it be preferrable to put even more of the parameters in `RunConfiguration`?